From: Michal Kubecek <mkubecek@suse.cz>
Date: Mon, 10 Oct 2016 01:14:35 +0200
Subject: kabi: hide name change of napi_gro_cb::udp_mark
Patch-mainline: Never, kabi workaround
References: bsc#1001486

The fix for bsc#1001486 renames member udp_mark of struct napi_gro_cb
to encap_mark as it's no longer used only for UDP but also for all
tunneling protocols. To preserve kABI, hide this change from genksyms.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 include/linux/netdevice.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index 9b6534f6e0b7..765795dfd9ad 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -1943,7 +1943,11 @@ struct napi_gro_cb {
 	u8	same_flow:1;
 
 	/* Used in tunnel GRO receive */
+#ifdef __GENKSYMS
+	u8	udp_mark:1;
+#else
 	u8	encap_mark:1;
+#endif
 
 	/* GRO checksum is valid */
 	u8	csum_valid:1;
-- 
2.10.0

